### DISCUSSION TITLE
2022/05/21/simple-access-management-with-sanity/

### DISCUSSION THREAD

**Discussion by giscus** [KittyGiraudel/site#163] [Announcements]
# 2022/05/21/simple-access-management-with-sanity/

I’m Kitty Giraudel, a non-binary trans frontend developer based in Berlin, focused on accessibility and inclusivity.

https://kittygiraudel.com/2022/05/21/simple-access-management-with-sanity/

**Comment by stephenhmarsh** [comment]
thaaaaank you.  this is helping me implement something where, even with paid features, i need certain users to be "edit only" (no create).
